ui/packages/consul-ui/app/components/topology-metrics/index.js

```diff
@@ -26,7 +26,10 @@ export default class TopologyMetrics extends Component {
     // TODO we can make the configurable even before we have a full solution for
     // multi-DC forwarding for Prometheus so providers that are global for all
     // DCs like an external managed APM can still load in all DCs.
-    if (this.env.var('CONSUL_DATACENTER_LOCAL') != this.args.topology.get('Datacenter')) {
+    if (
+      this.env.var('CONSUL_DATACENTER_LOCAL') != this.args.topology.get('Datacenter') ||
+      this.args.service.Service.Kind === 'ingress-gateway'
```

Comment:
Whilst you are in here, could you change this to a strict not equals `!==` on line 30 and maybe change the RB's `.get` to use the functional `get()` that we use everywhere else